<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFinanceiro</class>
 <widget class="QWidget" name="WidgetFinanceiro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Fluxo de Caixa</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="WidgetFluxoCaixa" name="widgetFluxoCaixa" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Contas a Pagar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="WidgetPagamento" name="widgetPagar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Contas a Receber</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="WidgetPagamento" name="widgetReceber" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Receber Resumo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="WidgetReceberResumo" name="widgetReceberResumo" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Vendas</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="WidgetVenda" name="widgetVenda" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Compras</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="WidgetFinanceiroCompra" name="widgetCompra" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetVenda</class>
   <extends>QWidget</extends>
   <header>src/widgetvenda.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetFluxoCaixa</class>
   <extends>QWidget</extends>
   <header>src/widgetfluxocaixa.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPagamento</class>
   <extends>QWidget</extends>
   <header>src/widgetpagamento.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetReceberResumo</class>
   <extends>QWidget</extends>
   <header>src/widgetreceberresumo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetFinanceiroCompra</class>
   <extends>QWidget</extends>
   <header>src/widgetfinanceirocompra.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
